### DISCUSSION TITLE
Alpine collapse plugin fast double click makes content appear over parent element

### DISCUSSION THREAD

**Discussion by Jumaniyozov** [alpinejs/alpine#2334] [7. Bugs (archived)]
Hi I found a bug while using collapse plugin if u click element multiple times without waiting for transition happen fully it makes content appear over the parent element.

Just click example button fast 
https://alpinejs.dev/plugins/collapse

**Comment by SimoTod** [comment]
I believe it's a side effect of #2213 cc @georgeboot

**Reply by georgeboot** [reply]
I'll look at it in the morning.
